packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -112,23 +112,33 @@ export interface BehavioralEventContext {
   };
 }
 
-export interface BehavioralEvent {
-  context: BehavioralEventContext;
+export type EventPayload = Record<string, string | number | boolean>;
```

Comment:
i know this seems a bit odd, and it's kind of a thought - but I think we could imagine sending some different types of payloads here, perhaps we should create a facade for the event Payload type which would go in the Record, and use that. That way if we need to change anything, we won't have to update the string | number | boolean in numerous locations, but instead, just update the internal type and it will propagate

Reply:
Sure - so you think about something in that spirit?

```
type Payload = string | number | boolean [ | something-else-later ]
export type EventPayload = Record<string, Payload>
```

Reply:
(I'm not 100% sure if extending the type here would make sense as we depends of the backend eventually, but I get your point from a coding engineering perspective)

Reply:
yes that's what I was imagining.